packages/EasyCore/src/Bridge/Symfony/ApiPlatform/DataTransformer/AbstractInputDataTransformer.php

```diff
@@ -29,12 +29,23 @@ public function setValidator(ValidatorInterface $validator): self
     public function supportsTransformation(mixed $data, string $to, ?array $context = null): bool
     {
         $apiResourceClass = $this->getApiResourceClass();
```

Comment:
Can we rename `getApiResourceClass` -> `getApiResourceClasses` and always iterate over array?

Reply:
In this way we can break BC with the other projects code which is using old method name.

Reply:
And for the same reason we can't always iterate over array because some code can currently return string in the implementation.

Reply:
Yeah, I see. The we shouldn't proceed with this solution, it has confusing contract.

Reply:
My opinion that nothing bad in such small BC break here, so we can rename and expect that this method always returns an array of classes.
The main thing is not to forget to add a BC break description to PR then.